packages/i18n/src/messages/fr.json

```
@@ -2277,15 +2282,38 @@
     "callControlsScreenshare": "Partage d’écran",
     "callControlsScreenshareActiveAria": "Partage d’écran, actif",
     "callControlsScreenshareInactiveAria": "Partage d’écran, inactif",
+    "callReactButton": "Réagir",
```

Biome (2.4.16)

[error] The key callReactButton was already declared.

(lint/suspicious/noDuplicateObjectKeys)